Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.19-alpine3.16 as builder
+
+WORKDIR /src
+
+# pre-copy/cache go.mod for pre-downloading dependencies and only redownloading them in subsequent builds if they change
+COPY go.mod go.sum ./
+RUN go mod download && go mod verify
+
+COPY . ./
+
+RUN go build -o /jobs-handler .
```

Comment:
🤔 will this be in `/` or in `/src`?
I am of the preference to have this on a dedicated folder, rather than just `/` root. `/build` maybe? `/src` also works although this is more of `/build` thing.

If I SSH into the container, I tend to feel that having `/build/jobs-handler` makes sense. Or something like that, WDYT?

Reply:
this is a multi-stage build so the only thing that will end up in the published image is everything form line 13 forward. We use this first container with the go tooling to build the cli and then just copy over the binary into the final container. In that sense the path we use here is not really important IMO other than making sure we use the same path in line 17

Reply:
We usually use a standard name `/app` folder so that all docker images are the same, no need to customise with `jobs-handler` per project.

Reply:
I actually like this idea better

Reply:
`/app` sounds good too, I'll use that